tests/cases/fourslash/importNameCodeFixNewImportTypeRoots0.ts

```diff
@@ -1,22 +0,0 @@
-/// <reference path="fourslash.ts" />

```

Comment:
`typeRoots` doesn't affect module resolution. It affects what directories are searched for automatically adding global definitions to the project. So it doesn't make sense to synthesize a specifier based on `typeRoots` because that will then be a compile error. `paths` should be used instead (which we already have a test for in `importNameCodeFix_fromPathMapping.ts`).
(Ref: Microsoft/TypeScript-Handbook#692)

Reply:
I dont agree with this, many times the typeRoots will add ambient module definitions to program and getting import module fix would be good. Adding @DanielRosenwasser please comment on this. (look at vscode code i believe it adds typeRoots to add additional typings)

In anycase instead of deleting this test case (where we are using typeroots, add negative test to help catch any future regressions)

Reply:
If there is an ambient module definition we should already be handling that without typeRoots. See `importNameCodeFixNewImportAmbient0.ts`.